packages/mdc-tabs/tab-bar-scroller/foundation.js

```diff
@@ -144,16 +144,16 @@ export default class MDCTabBarScrollerFoundation extends MDCFoundation {
 
     for (let i = 0; i < this.adapter_.getNumberOfTabs(); i++) {
       const tabOffsetLeftAndWidth =
-        this.adapter_.getComputedLeftForTabAtIndex(i) + this.adapter_.getComputedWidthForTabAtIndex(i);
+          this.adapter_.getComputedLeftForTabAtIndex(i) + this.adapter_.getComputedWidthForTabAtIndex(i);
```

Comment:
I'm not sure why there are indentation changes in this commit, but they seem unrelated to what was actually changed. Can you revert them?